doc/production.rst

```diff
@@ -74,3 +83,51 @@ tag you want to install on the machines:
    argument from both the ``discos-deploy`` and ``discos-get`` scripts. If you
    pass the ``--station`` argument anyway, if the given argument does not match
    the correct station you will receive an error and the procedure will stop.
+
+Replace the Manager in case of failure
+--------------------------------------
+In case the Manager machine suffers a failure of some sort, it has to be
+replaced. In order to do this, the first thing to do is perform again the
```

Comment:
"is perform" or "is to perform" ?

Reply:
I'm investigating this with an English speaking friend, I'll post the correct version ASAP

Reply:
About the point below, it is not clear what are "all station systems".